kinto/plugins/accounts/views.py

```diff
@@ -138,3 +223,192 @@ def on_account_changed(event):
     cache_key = utils.hmac_digest(hmac_secret, ACCOUNT_CACHE_KEY.format(username))
     # Delete cache
     cache.delete(cache_key)
+
+
+# Account validation (enable in the settings).
+validation = Service(
+    name="account-validation",
+    path="/accounts/{user_id}/validate/{activation_key}",
+    description="Validate an account",
+)
```

Comment:
Quick note here: the `validation` and the `reset-password` endpoint (services) are available, even if the account validation option isn't enabled in the settings. However, they would not work properly. I'm not sure how to conditionally add a Service?

Reply:
Yep that would be great: https://github.com/Kinto/kinto-fxa/blob/master/kinto_fxa/__init__.py#L65-L68